niworkflows/func/timeseries1D.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+
+"""
+Workflows for applying image processing tools to TSV files
+"""
+
+from nipype.interfaces import utility as niu, fsl
+import nipype.pipeline.engine as pe
+from ..interfaces.timeseries1D import (
+    TSVToImage, ImageToTSV, TSVFrom1D, TSVTo1D
+)
+
+
+def init_apply_to_2d_wf(function_node,
+                        t_rep,
+                        name='apply_to_2d_wf',
+                        mode=None,
+                        input_field='in_file',
+                        output_field='out_file'):
+    """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First word of the docstring should not be `This`](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096913)